src/components/molecules/Menu/Menu.jsx

```diff
@@ -37,7 +54,7 @@ const Menu = ({ menu, showMenu, openSubMenu }) => {
     }
   };
   return (
-    <div className={showMenu}>
+    <div className="show" ref = {menuRef}>
```

Comment:
Class name show seems like a weird class name to me. What are you using it for?

Reply:
Changed to "menu"